src/blocktree_processor.rs

```diff
@@ -439,19 +447,20 @@ mod tests {
         );
 
         // Now ensure the TX is accepted despite pointing to the ID of an empty entry.
-        par_process_entries(&bank, &[entry]).unwrap();
+        par_process_entries(&bank, &slot_entries).unwrap();
         assert_eq!(bank.process_transaction(&tx), Ok(()));
     }
 
     #[test]
     fn test_process_ledger_simple() {
+        solana_logger::setup();
         let leader_pubkey = Keypair::new().pubkey();
         let (genesis_block, mint_keypair) = GenesisBlock::new_with_leader(100, leader_pubkey, 50);
-        let (ledger_path, last_id) = create_new_tmp_ledger!(&genesis_block);
+        let (ledger_path, mut last_entry_hash) = create_new_tmp_ledger!(&genesis_block);
         debug!("ledger_path: {:?}", ledger_path);
 
         let mut entries = vec![];
-        let mut last_entry_hash = last_id;
+        let last_id = genesis_block.hash();
```

Comment:
I'm a bit confused on the few places in this PR that still have `last_id`. Rebase error?

Reply:
Oh.  I'll do the full `last_id` renaming in a separate PR, that'll be a mechanical change that nobody wants to review.   This PR is the 🍖 that actually needs review, so tried to keep it as small and tight as possible